boards/others/esp32c3_supermini/board.yml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+board:
+  name: esp32c3_supermini
+  vendor: espressif
```

Comment:
Please change the vendor name

Reply:
You can use `others` as vendor name here.